### PR TITLE
SortedMembers::add for pallet-membership benchmarks

### PR DESCRIPTION
**Pull request by Dinonard**
Adds implementation for `SortedMembers::add` for _pallet-membership_ benchmarks.

### CODE REVIEW

**Review by ggwpez — APPROVED** [review]

**Comment by Dinonard** [comment]
Can we please get another approval? 🙂

**Comment by Dinonard** [comment]
Please someone provide another approval (or comment). Thanks!

**Comment by ggwpez** [comment]
Reviewing changes here is currently mainly done by Parity, but it does not scale. Will ping it in the [Polkadot Technical Fellowship - Open Channel](https://app.schildi.chat/#/room/#fellowship-open-channel:parity.io).

**Review by wirednkod — APPROVED** [review]

File: substrate/frame/membership/src/lib.rs [hunk 1]
```diff
@@ -369,6 +369,17 @@ impl<T: Config<I>, I: 'static> SortedMembers<T::AccountId> for Pallet<T, I> {
...
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn add(new_member: &T::AccountId) {
+		use frame_support::{assert_ok, traits::EnsureOrigin};
+		let new_member_lookup = T::Lookup::unlookup(new_member.clone());
+
+		assert_ok!(Pallet::<T, I>::add_member(
+			T::AddOrigin::try_successful_origin().unwrap(),
+			new_member_lookup,
+		));
```
Comment (bkchr) [inline substrate/frame/membership/src/lib.rs:381]:
```suggestion
		if let Ok(origin) = T::AddOrigin::try_successful_origin() {
			assert_ok!(Pallet::<T, I>::add_member(
				origin,
				new_member_lookup,
			));
		} else {
			log::error!(target: LOG_TARGET, "Failed to add `{new_member:?}` in `SortedMembers::add`.")
		}
```

**Review by bkchr — APPROVED** [review]

**Comment by ggwpez** [comment]
bot fmt

**Pull request merged**